docs/04-resources/bucket.md

```diff
@@ -79,7 +79,8 @@ store.onCreate(inflight (key: str) => {
 });
 
 store.onDelete(inflight (key: str) => {
-  log("Deleted " + key);
+  copies.delete(key);
```

Comment:
You need to make your changes on https://github.com/winglang/wing/blob/main/libs/wingsdk/src/cloud/bucket.md and the SDK mutation job will copy them here